docs/CPS/CPS-XXXX/README.md

```diff
@@ -0,0 +1,113 @@
+- A custom schema was implemented due to the lack of a standard
+- 
+> [!NOTE]
+> Since this API was developed, [Cardano DB Sync] has since added a mode to store transactions in CBOR, but at the time of implementation it was not an option.
+
+#### Current Alternatives
+[Blockfrost WebSocket Link] is a partial solution for the impedance mismatch problem, however it's not addressing the fragility and standards problem. It's not practical to run this service to satisfy a single-tenant, full node, use case. 
+
+## Goals
+- Define a standard API that is optimal for fundamental wallet operation. Consider extensibility for optional enhancement.
+- Deliver data as a filtered subset of the network state, using native encoding formats, to simplify server implementations and push processing to the edge of the network.
+- Foster long term stability of the API when crossing major Cardano protocol upgrade boundaries.
+- Facilitate event-driven wallet behavior, minimising the need for long-polling, and to increase the sophistication of infrastructure management and user-base insights.
+- Support script-based wallets
+- Minimise the network demand by allowing the server to control the throttling strategy, rather than a pull-based approach like we see in the local chain-sync protocol. 
+- API can be applied in a local, single-tenant environment, and a remote multi-tenant deployment, to facilitate a universal approach to wallet supporting an optional full local node for power users. 
+- Remove the need for wallets to perform address discovery
+- Make it possible for a user to quickly derive new accounts and their addresses without re-projecting from genesis. 
+
+## Open Questions
+- What identifiers will wallets provide for the server to filter on?
+- What is the most efficient solution for indexing data based on account identifiers?
+- Will it support multiple accounts?
+- How will chain rollbacks be handled?
+- What design considerations are made to establish a stable API over network protocol upgrades?
+- Will the API standard support extensions?
+- Should verification of stake and voting rights delegation be considered part of fundamental wallet behaviour or an extension?
+- Can Mithril be used to verify state snapshots for provided account credentials, to facilitate a fast catchup feature? If not, what is a fallback approach for resource and/or network-constrained applications?
+- How will the API behaviour differ for single-tenant use cases?
+- Are there any natural authentication strategies?
+- Should the Block be transmitted to wallet, similar to the Ouroboros Chain-Sync protocol, or just the relevant transactions?   
+- Will this API be applicable for other Blockchains?
+- What is the source of rewards data, given there is no node API currently?
+- What is the most efficient strategy to prepare the database for single-tenant use-cases?
```

Comment:
This seems like a client-side concern, while the rest of the CPS is about a backend API

Reply:
The database is a server concern, but I agree this is more of an implementation concern. Will remove

Reply:
https://github.com/will-break-it/wallet-architecture/pull/3/commits/27802ddb1577fd1fc8657421ec6fb7ed01d1c058

Reply:
https://github.com/will-break-it/wallet-architecture/pull/3/commits/7b7ca0ce64ace467ca8e09a0b7d722c9f91e371d

Reply:
https://github.com/will-break-it/wallet-architecture/commit/27802ddb1577fd1fc8657421ec6fb7ed01d1c058